.github/workflows/build.yml

```diff
@@ -104,20 +187,46 @@ jobs:
       - name: Build archive
         run: |
           # not sure why we're building this one
-          # mv aarch64-unknown-linux-gnu.so jniLibs/arm64-v8a/libbindings_ffi.so
+          # mv aarch64-unknown-linux-gnu/libbindings_ffi.so jniLibs/arm64-v8a/
```

Comment:
This is the fix for the mispackaged libxmtp-android.zip. I didn't realize that the artifact downloads always create a directory named after the artifact.